source/WindowScroller/WindowScroller.js

```diff
@@ -95,6 +102,18 @@ export default class WindowScroller extends Component {
     onResize({ height })
   }
 
+  _enablePointerEventsAfterDelay () {
+    if (this._disablePointerEventsTimeoutId) {
+      clearTimeout(this._disablePointerEventsTimeoutId)
+    }
+
+    this._disablePointerEventsTimeoutId = setTimeout(() => {
```

Comment:
In the spirit of "everything counts" I've been trying to avoid creating throw-away functions for things that happen frequently (like scrolling). For instance I recently updated this same bit of code in `Grid` to reuse a single function:
https://github.com/bvaughn/react-virtualized/blob/master/source/Grid/Grid.js#L604 n

I think we should do the same here.